consensus/spos/worker.go

```diff
@@ -377,6 +382,16 @@ func (wrk *Worker) ProcessReceivedMessage(message p2p.MessageP2P, _ func(buffToS
 	return nil
 }
 
+func (wrk *Worker) processReceivedHeaderMetric(cnsDta *consensus.Message) {
+	if !wrk.consensusState.IsNodeLeaderInCurrentRound(string(cnsDta.PubKey)) {
```

Comment:
If the node is not synced this condition would return unexpected result

Reply:
Agree, when it will happen that sync-ed block's leader == current round leader it will compute the received block relative time position. When syncing, the metric will update at large intervals because of the above mentioned reason.